### PR TITLE
Fix/enable updating rush events

### PR DESCRIPTION
**Pull request by wderocco8**
## What does this PR do?

- allowed for editing `rush-event` objects
- updated UI for event Cards (to use Flowbite)
- added `deadline` and `lastModified` fields
    - replaced `dateCreated` temporarily (can add back if needed)

## Type of change

- [x] Fix: Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor: Any code refactoring
- [ ] Chore: technical debt, workflow improvements
- [ ] Feature: New feature (non-breaking change which adds functionality)
- [ ] Documentation: This change requires a documentation update

## Screenshots

https://drive.google.com/file/d/1KsSRRrYDHH9ZSQr1N8Mi5nLdM0w2uxy3/view?pli=1

### CODE REVIEW

File: app/admin/rush/page.tsx [hunk 1]
```diff
@@ -83,15 +112,49 @@ export default function RushEvents() {
...
+                    Created:
+                    <MongoTimestamp datestring={event.dateCreated} />
+                  </p> */}
+                  <p className="flex gap-2 truncate text-sm text-gray-500 dark:text-gray-400 mr-1">
+                    Last Modified:
+                    <MongoTimestamp datestring={event.lastModified} />
+                  </p>
+                  <p className="flex gap-2 truncate text-sm text-gray-500 dark:text-gray-400 mr-1">
+                    Deadline:
+                    <MongoTimestamp datestring={event.deadline} />
```
Comment (jinyoungbang) [inline app/admin/rush/page.tsx:126]: Maybe just calling the library itself makes more sense - I saw the code for the wrapper and it just essentially does the same / unless we have plans to make specific timestamps, I think replacing it with the library and deleting the mongotimestamp component makes more sense!

File: components/admin/events/EventModal.tsx
Comment (jinyoungbang) [inline components/admin/events/EventModal.tsx:1]: do you mind fixing the indents to two spaces?

**Review by jinyoungbang — APPROVED** [review]
LGTM! Please address the comments and feel free to merge 👍

File: app/admin/rush/page.tsx (hunk 1 above)
Comment (wderocco8) [reply app/admin/rush/page.tsx:126]: agreed, I went back and forth on that lol

File: components/admin/events/EventModal.tsx
Comment (wderocco8) [reply components/admin/events/EventModal.tsx:1]: yuh

**Pull request merged**